Add NASA certification process (#926)
open-life-science/open-life-science.github.io: nebula/nebula-cohort-procedures/nasa-certification.md

---
layout: page
title: NASA TOPS Certification Process
image: https://images.unsplash.com/photo-1526212076326-45d0c30a89e8
photos:
  name: Numendil
  license: CC-BY
  url: https://images.unsplash.com/photo-1526212076326-45d0c30a89e8
---

**Congratulations!** Upon graduating from [Nebula](https://we-are-ols.org/nebula/) or [Open Seeds](https://we-are-ols.org/openseeds/), you will earn a NASA TOPS Badge via [Credly](https://www.credly.com/organizations/nasa-tops/badges) certifying your knowledge of Open Science best practices. Please follow these steps to finalize the certification process. 

## Steps to claim your NASA certificate

1. **Create an [ORCID](https://orcid.org/) account** if you don't have one already. 

> 2. **Create and log into an account on https://openscience101.org/.*** 

Reviewer: ```suggestion
2. **Create and log into an account on [openscience101.org](https://openscience101.org/).** 

```

## Steps to claim your NASA certificate

1. **Create an [ORCID](https://orcid.org/) account** if you don't have one already. 

2. **Create and log into an account on [openscience101.org](https://openscience101.org/).** 

- You must use the same email as your ORCID. This step links your badge to your ORCID. 
- There is **no need** to complete the MOOC or assessments. 

3. **Complete the post-cohort feedback survey.**
- The OLS team will provide a link to the feedback survey after the cohort ends. 
- You will need to submit the following information for your certificate: full name, ORCID, and the email associated with your ORCID. 

4. **Accept your NASA badge.**
- Within 30 days of submitting your information, you will receive an email from Credly inviting you to accept your digital badge. 
- Once accepted, the badge will appear in your ORCID account.

### Contact

> For any questions about this process, please contact us at nebula-cohort-management@we-are-ols.org.

Reviewer: ```suggestion
For any questions about this process, please contact us at [{{ site.nebula_email }}@{{ site.email_extension }}](mailto:{{ site.nebula_email }}@{{ site.email_extension }}).

```
